composer.json

```diff
@@ -15,10 +15,12 @@
         "php": ">=5.3.2"
     },
     "require-dev": {
-        "react/event-loop": "0.3.*@dev"
+        "react/event-loop": "0.3.*@dev",
+        "react/promise": "~2.2"
     },
     "suggest": {
-        "react/event-loop": "For using event-loop based scheduling."
+        "react/event-loop": "For using event-loop based scheduling.",
+        "react/promise": "For converting promises to and from observables."
```

Comment:
Let's move this discussion to the pull request I created on @davidwdan's repository. I did the work there: https://github.com/davidwdan/Rx.PHP/pull/2.